src/pages/IndividualArticlePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { getIndividualArticle } from '../utilsAPI';
+import { useEffect, useState } from 'react';
+
+const IndividualArticlePage = () => {
+  const { article_id } = useParams();
+  const [articleData, setArticleData] = useState([]);
+
+  useEffect(() => {
+    getIndividualArticle(article_id).then(({ data }) => {
+      console.log(data.article[0]);
+      setArticleData(data.article[0]);
```

Comment:
It might be nice to return just the individual article in the API function to keep the component clear but thats completely up to you :)